components/raftstore-v2/Cargo.toml

```diff
@@ -32,18 +32,24 @@ cloud-azure = ["raftstore/cloud-azure"]
 batch-system = { path = "../batch-system", default-features = false }
 collections = { path = "../collections" }
 crossbeam = "0.8"
+engine_rocks = { path = "../engine_rocks", default-features = false }
```

Comment:
You should not make it as a dependency as we are supposed to generic over engine trait.